modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+
+[id="installation-osp-accessing-api-floating_{context}"]
+
+= Enabling API access with floating IP addresses
+
+Make OpenShift API endpoints accessible by attaching floating IP (FIP) addresses to them. You create two floating IP addresses: one to attach to the API load balancer (`lb FIP`) and one to attach to OpenShift applications (`apps FIP`).
+
+[IMPORTANT]
+The load balancer FIP is the address used in the `install-config.yaml` file.
```

Comment:
In the rendered doc, this section appears *after* running the installer. If someone runs the installer without having a floating IP it will fail and they won't know why (sure, they should read the whole thing but still).

I think it's totally okay to leave the section here, but we should probably add the `openstack floating ip create` command before running the installer too.

Reply:
I'll relocate this into the the prereqs and prep part of the document.